packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -296,9 +310,9 @@ class ValidatorsList extends React.PureComponent<ValidatorsListProps & I18nProps
               />
             </View>
             {validatorGroups.map((group, i) => (
-              <View key={group.address} onClick={this.expand.bind(this, i)}>
+              <div key={group.id} onClick={this.expand.bind(this, i)}>
```

Comment:
why this change to a div?

Reply:
Because there was a lot of issues related with the z-index because of View default styling